frontend/src/components/transactions/ReceiptsOutcomeList.tsx

```diff
@@ -0,0 +1,38 @@
+import * as T from "../../libraries/explorer-wamp/transactions";
+import ReceiptOutcomeRow from "./ReceiptOutcomeRow";
```

Comment:
It seems that we display the full Receipt information, not only outcomes, so let's name these components as `ReceiptsList`, `ReceiptRow`, etc (leave out "outcome" unless you feel it reflects the situation.